src/OrchardCore.Modules/OrchardCore.Menu/Views/MenuItemLink-LinkMenuItem.cshtml

```diff
@@ -1,7 +1,10 @@
 @using OrchardCore.ContentManagement
-
 @{
     ContentItem contentItem = Model.ContentItem;
     var linkMenuItemPart = contentItem.As<LinkMenuItemPart>();
+
+    TagBuilder tag = Tag(Model, "a");
+    tag.Attributes["href"] = (string)linkMenuItemPart.Url;
+    tag.InnerHtml.Append((string)linkMenuItemPart.Name);
 }
-<a href="@Url.Content(linkMenuItemPart.Url)">@linkMenuItemPart.Name</a>
+@tag
```

Comment:
Don't remove to EOL at the end of the files

Reply:
Put it back in.